compiler/scopelang/ast.mli

```diff
@@ -164,3 +169,13 @@ val make_let_in :
   expr Pos.marked Bindlib.box ->
   expr Pos.marked Bindlib.box ->
   expr Pos.marked Bindlib.box
+
+val make_default :
+  ?pos:Pos.t ->
+  expr Pos.marked list ->
+  expr Pos.marked ->
+  expr Pos.marked ->
+  expr Pos.marked
+(** [make_default ?pos exceptions just cons] builds the equivalent to an
+    [EDefault] term, while avoiding redundant nested constructions. The position
+    is extracted from [just] by default. *)
```

Comment:
The fact that the position is extracted from `just` is a printing trick that will play nicely with the error messages raised when two exceptions conflict with each other : the error message will print the positions of the conflicting justifications.